Bryn Mawr Classical Review.js

```diff
@@ -134,23 +154,13 @@ var testCases = [
 				"itemType": "journalArticle",
 				"creators": [
 					{
-						"firstName": "Christina S.",
-						"lastName": "Kraus",
-						"creatorType": "author"
-					},
-					{
-						"firstName": "Anthony",
-						"lastName": "Grafton",
-						"creatorType": "reviewedAuthor"
-					},
-					{
-						"firstName": "Glenn W.",
-						"lastName": "Most",
-						"creatorType": "reviewedAuthor"
+ 						"firstName": "Keely Elizabeth",
+ 						"lastName": "Heuer",
+ 						"creatorType": "author"
 					},
 					{
-						"firstName": "Salvatore",
-						"lastName": "Settis",
+						"firstName": "Rolf",
+						"lastName": "Hurschmann",
```

Comment:
>Yes, the second test case is 04, not 44. It looks like the original test case had both 04 and 44.

Where did it have both? All I see is 44 and all the metadata looks right (except for the date). Am I missing something?